src/lib/components/cards/MemberCardTag.svelte

```diff
@@ -1,20 +1,29 @@
 <script lang="ts">
     import Icon from '@iconify/svelte';
 
-    import type { Member } from '$lib/models/member';
+    import type { Position } from '$lib/models/position';
+
     import { getSocialMedium } from '$lib/types/social_media';
 
     interface Props {
-        member: Member;
+        name: string;
+        title?: Position[];
+        socials?: Record<string, string>;
     }
 
-    const { member }: Props = $props();
-    const { name, title, socials } = $derived(member);
+    const { name, title, socials }: Props = $props();
 </script>
 
 <div class="*:!m-0">
     <p class="text-md md:text-lg">{name}</p>
-    <p class="text-xs leading-tight">{title}</p>
+    {#if title}
+        <p class="text-xs leading-tight">
+            {#each title.slice(0, -1) as pos}
+                {pos},
+            {/each}
+            {title.slice(-1)}
```

Comment:
Whoops, my comment for this got removed. Anyways, I suggest using `.join(...)` here instead.

```suggestion
            {title.join(", ")}
```